kani-compiler/src/codegen_cprover_gotoc/context/goto_ctx.rs

```diff
@@ -283,13 +289,14 @@ impl<'tcx> GotocCtx<'tcx> {
     pub fn register_initializer(&mut self, var_name: &str, body: Stmt) -> &Symbol {
         let fn_name = Self::initializer_fn_name(var_name);
         let pretty_name = format!("{var_name}::init");
+        let loc = *body.location();
         self.ensure(&fn_name, |_tcx, _| {
             Symbol::function(
                 &fn_name,
                 Type::code(vec![], Type::constructor()),
-                Some(Stmt::block(vec![body], Location::none())), //TODO is this block needed?
+                Some(Stmt::block(vec![body], loc)), //TODO is this block needed?
```

Comment:
Not your fault, but this comment is odd. Do we have a GitHub issue to place it here?

Reply:
Reviewed all uses, it's actually indeed not necessary. Added an assertion to make sure this remains ok.